org.putty.putty.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.putty.putty
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+rename-desktop-file: putty.desktop
+rename-icon: putty
+command: putty
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --filesystem=/run/udev:ro
+modules:
+  - name: putty
+    buildsystem: cmake-ninja
+    build-options:
+      cflags: -DNOT_X_WINDOWS -Wno-error=unused-function
+    post-install:
+      - mkdir -p ${FLATPAK_DEST}/share/applications/ ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/ ${FLATPAK_DEST}/share/metainfo/
+      - install -Dm755 putty.desktop ${FLATPAK_DEST}/share/applications/
+      - install -Dm644 putty.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
+      - install -Dm644 org.putty.putty.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
```

Comment:
You can use `install`'s option `-t` to set destination directory instead of creating it manually.
```suggestion
      - install -Dm755 putty.desktop -t ${FLATPAK_DEST}/share/applications/
      - install -Dm644 putty.svg -t ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
      - install -Dm644 org.putty.putty.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/
```

Reply:
Thanks for the tip!